docs/paper/dev/misc/reading-stacktraces.md

```diff
@@ -0,0 +1,51 @@
+# Reading Stacktraces
```

Comment:
Is it worth generalizing this, or dumbing it down a little and putting it not in the dev docs, but somewhere else? It's useful for non-devs to understand the very basics so they can maybe identify some causes if its plugin or something they can handle themselves.

Reply:
"Reading Stacktraces/Plugin Errors"? I'd still include "stacktrace" in the title

Reply:
I wasn't referring to the name, but the location of this inside the `dev/` docs section.